spec/system/group_index_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe 'Groups', type: :system do
+  include Devise::Test::IntegrationHelpers
+
+  before do
+    @user = User.create(name: 'muneeb', email: 'muneeb@gmail.com', password: '123456')
+    login_as(@user, scope: :user)
+    @group = Group.create(name: 'School',
+                          icon: 'https://example.jpg', user: @user)
+  end
+
+  describe 'GET index' do
+    it 'displays  categories information ' do
+      visit user_groups_path(user_id: @user.id)
+
+      expect(page).to have_css('img[src="https://example.jpg"]')
+      expect(page).to have_content(@group.name)
+    end
+
+    it 'displays "Add new group" link' do
+      visit user_groups_path(user_id: @user.id)
+      expect(page).to have_text('Add new')
+    end
+
+    it 'displays "create" link to create new category' do
+      visit new_user_group_path(user_id: @user.id)
+      expect(page).to have_text('create')
+    end
+
+    it 'displays total amount ' do
+      visit user_groups_path(user_id: @user.id)
+      expect(page).to have_text('$ 0')
```

Comment:
- [ ] `Required`
- [ ] Kindly fix all the failing tests
- [ ] please screenshot below
- [ ] 
![image](https://github.com/muneeb3123/Budget-App/assets/66230306/37097626-972d-4f26-a077-5ec0f773681e)

Reply:
in my case its working properly i attached video see

https://github.com/muneeb3123/Budget-App/assets/118625824/b5ea0b57-99f3-4aa1-a753-efd532353cbe

Reply:
@muneeb3123 Can we do a zoom call so i show you? and you also show me?

Reply:
@mwanawabangona    
ok sure